<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFieldWindow</class>
 <widget class="QDialog" name="NewFieldWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Campo</string>
  </property>
  <widget class="QLineEdit" name="NFWLe_Name">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="NFWCB_Type">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Cadena</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Entero</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Real</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="NFWSp_Length">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="NFWSp_Decimal">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="NFWCB_Key">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Activar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NFWBt_add">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitud:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>150</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Espacios Decimales:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Llave Primaria:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
